schemas/json-with-comments.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Just to confirm.
The existence of this file is to allow trailing commas in `lsp-json-schemas.json` and `lsp-json-schemas_extra.json `?

Reply:
Yes, it's a generic schema that only allows commas and comments and doesn't define any specific schema otherwise.